StortSpelProjekt/Game/src/Gamefiles/UpgradeManager.h

```diff
@@ -1,20 +1,20 @@
 #ifndef UPGRADE_MANAGER_H
 #define UPGRADE_MANAGER_H
-
-#include "Components/UpgradeComponents/UpgradeComponent.h"
 #include "Components/UpgradeComponents/Upgrades/Upgrade.h"
```

Comment:
This can also be forward declared. No object of this type is used in the header, only pointers